api.ts

```diff
@@ -90,3 +96,95 @@ interface Status {
     nanos: number;
   };
 }
+
+export interface BlockShallow {
+  height: number;
+  /**
+   * ISO DateTime
+   */
+  timestamp: string;
```

Comment:
Should this be stored as a `BigNum` instead? We use a `u128` to store the timestamp,